### ISSUE TITLE
Investigate if there’s a possibility to harmony patch B9PS without depending on it.

### ISSUE THREAD

**Issue by SofieBrink** [KSPModdingLibs/KSPCommunityPartModules#12]
- it’d be nice to have the ability to add a B9PS module that’s switchable in flight to the action groups. There’s a no good implementation here in [SEP](https://github.com/Kari1407/Starship-Expansion-Project/blob/main/Source/Modules/ModuleSEPPartSwitchAction.cs) but that uses a part module and also means we depend on B9PS which isn’t ideal
